Corona-Warn-App/src/main/java/de/rki/coronawarnapp/exception/reporting/ErrorReportReceiver.kt

```diff
@@ -52,31 +47,44 @@ class ErrorReportReceiver(private val activity: Activity) : BroadcastReceiver()
             message += "#$apiStatusCode"
         }
 
-        val errorTitle = context.resources.getString(R.string.errors_generic_details_headline)
-            .toUpperCase(Locale.ROOT)
-
-        if (CoronaWarnApplication.isAppInForeground) {
-            DialogHelper.showDialog(
-                DialogHelper.DialogInstance(
-                    activity,
-                    "$errorTitle: $errorCode\n$title",
-                    message,
-                    confirm,
-                    details,
-                    null,
-                    {},
-                    {
-                        DialogHelper.showDialog(
-                            DialogHelper.DialogInstance(
-                                activity,
-                                title,
-                                "$detailsTitle:\n$stack",
-                                confirm
-                            )
-                        ).run {}
-                    }
-                ))
+        val dialogTitle = if (intent.hasExtra(ReportingConstants.ERROR_REPORT_TITLE_EXTRA)) {
```

Comment:
Minor: could be moved to a dedicated function in order to get rid of the @Suppress("LongMethod")

Reply:
I currently don't see an easy way to restructure the code due to overlapping use of properties (e.g. `title`). I'd prefer to not refactor it in this PR, the whole error reporting via intents could need an overhaul too :frowning: 

Would it be okay to leave it as is?

Reply:
Sure, I don't expect minors to be fixed.